tutorials/networking/http_client_class.rst

```diff
@@ -10,14 +10,22 @@ which has a tutorial available :ref:`here <doc_http_request_class>`.
 Here's an example of using the :ref:`HTTPClient <class_HTTPClient>`
 class. It's just a script, so it can be run by executing:
 
-.. code-block:: console
+.. tabs::
 
+ .. code-tab:: console GDScript
+ 
     c:\godot> godot -s http_test.gd
-
+	
+ .. code-tab:: console C#
+ 
+	c:\godot> godot -s HTTPTest.cs
```

Comment:
Indentation issue, code should be indented with 4 spaces per level (same for the C# code below).

Reply:
Sorry if this is a dumb question, but this is my first time contributing to the documentation and I'm not sure what I'm supposed to do here - Should I make another commit to fix the indentation, close this PR and open a new one with only one commit so the commit history doesn't get cluttered up, or have you fixed it already?

Reply:
@TechnoPorg In the docs particularly we don't mind multiple commits much, but in general you'd just make a new commit locally, then squash them all together via rebase, and then force push your new single commit to the same branch. This PR will be updated.

We recommend and detail this process in our [contribution guide](https://docs.godotengine.org/en/stable/community/contributing/pr_workflow.html).